### ISSUE TITLE
Deserialized field type cannot contain namespace

### ISSUE THREAD

**Issue by DmitrySamoylov** [media-io/yaserde#24]
Hey everybody.
Compilation fails if I specify field name along with namespace. Without namespace `std::string::` everything is fine.

```rust
use std::io::Read;
use yaserde::YaDeserialize;
use yaserde_derive::YaDeserialize;
use log::debug;

#[derive(YaDeserialize, Debug)]
struct MyRoot {
    my_field: std::string::String,
}

fn main() {
    let config: MyRoot = yaserde::de::from_str("").unwrap();
    println!("{:#?}", config);
}
```

Error message:

```
error[E0063]: missing field `my_field` in initializer of `MyRoot`
 --> src/main.rs:7:8
  |
7 | struct MyRoot {
  |        ^^^^^^ missing `my_field`
```

**Comment by MarcAntoine-Arnaud** [comment]
Hello,

You can found some examples here:
https://github.com/media-io/yaserde_examples/blob/master/src/bbigras_namespace.rs

You've to define the `namespace`, and `prefix`.

Best regards,
Marc-Antoine

**Comment by DmitrySamoylov** [comment]
Yeah I know that yaserde supports XML namespaces. I mean that only Path with a single PathSegment is accepted as a field type. Use case for this are branchy structures spread over different modules:

```rust
#[derive(YaDeserialize)]
struct MyType {
    my_field: other_module::OtherType,
}
```

**Comment by MarcAntoine-Arnaud** [comment]
Hooo sorry I have bad understood.
Yes currently it's not supported.
I have to take a look on it.

**Comment by MarcAntoine-Arnaud** [comment]
Merged ;-)

**Comment by MarcAntoine-Arnaud** [comment]
published in the 0.3.8.
